capstone/src/main/webapp/style.css

```diff
@@ -1,8 +1,135 @@
+a {
+    color: #2c2c2c;
+}
+
+.bg {
+    background-color: #2c2c2c;
+}
+
+.bg-1 { 
+    background-color: #2c2c2c;
+    color: #ffffff;
+    padding-bottom: 30px;
+}
+
+.bg-4 {
+    background-color: #000000;
+    color: #ffffff;
+}
+
 body {
-  margin-left: auto;
-  margin-right: auto;
-  text-align: center;
-  width: 650px;
+    font-family: 'Vollkorn', serif;
+    line-height: 1.8;
+    color: #2c2c2c;
```

Comment:
Some classes still need the color property even though the default color on the html is #2c2c2c. I thought that #2c2c2c would be our default color and if specific things needed other colors you would create the necessary color attribute (rather than putting #2c2c2c in every class/id).